wallettemplate/build.gradle

```diff
@@ -20,7 +18,7 @@ javafx {
     modules = [ 'javafx.controls', 'javafx.fxml' ]
 }
 
-sourceCompatibility = 1.11
```

Comment:
I'm not so sure about this change. I know OpenJDK 11 supports this syntax, but I believe OpenJDK 8 does not. And I'd like to keep the type of syntax used consistent with the other subprojects.